src/Services/InteractsWithPage.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace Modelizer\Selenium\Services;
+
+trait InteractsWithPage
```

Comment:
I'm fine either way,

As I mentioned in my overall comment, I used plural on these to keep it "expressive" in terms of reading it.